cmsl1t/recalc/resolution.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+from exceptions import RuntimeError
+from math import pi, degrees
+twopi = 2 * pi
+
+
+def get_resolution_function(resolution_type):
+    if resolution_type.lower() == "energy":
```

Comment:
might be worth to cache `resolution_type.lower()`, i.e `rtype = resolution_type.lower()` and use that for comparison #speed